CHANGELOG.md

```diff
@@ -1,6 +1,7 @@
 # Changelog
 
 ## Draft
+- Performance improvements. [#1229](https://github.com/bigcommerce/cornerstone/pull/1229)
```

Comment:
there are a ton of changes in this release - where can theme developers to get more information on the 2.0 changes and how it might affect them? (besides the commits themselves)

CC @nikita-ahuja

Reply:
The PR has many more details, and I've also put together an expanded version that we can use to announce the release. In addition, we will be covering this in the upcoming town hall.

Reply:
thanks!

Reply:
Would it be better to itemize the various improvements with "Performance improvements:" as a prefix? They'd all end up linking to the same PR, though.